easybuild/easyblocks/g/gromacs.py

```diff
@@ -357,3 +368,25 @@ def sanity_check_step(self):
             'dirs': dirs,
         }
         super(EB_GROMACS, self).sanity_check_step(custom_paths=custom_paths)
+
+    def get_gromacs_arch(self, build_arch):
+        """Determine value of GMX_SIMD CMake flag based on optarch string.
+
+        Refs:
+        [0] http://manual.gromacs.org/documentation/2016.3/install-guide/index.html#typical-installation
+        [1] http://manual.gromacs.org/documentation/2016.3/install-guide/index.html#simd-support
+        [2] http://www.gromacs.org/Documentation/Acceleration_and_parallelization
+        """
+        build_arch = build_arch.upper()
+        if build_arch.endswith(("AVX2")) and not LooseVersion(self.version) < LooseVersion('5.0'):
```

Comment:
Does it need to *end* with `avx2`, or is having `avx2` anywhere in the list of options sufficient? If the latter, use  something like

```python
if 'AVX2' in build_arch:
```

Also, simplify the version check using positive logic (easier to understand):

```python
LooseVersion(self.version) >= LooseVersion('5.0')
```